security/ca-policy-egi-core.pan

```diff
@@ -1,115 +1,114 @@
 #
-# @(#)ca-policy-egi-core.tpl 1.60-1
-# Copyright (c) Stichting European Grid Initiative, 2014
+# @(#)ca-policy-egi-core.tpl 1.73-1
+#
+# Copyright (c) Stichting EGI, 2016
+#
 # This work is licensed under the Creative Commons Attribution-Noncommercial
 # 3.0 License. To view a copy of this license, visit
 #   http://creativecommons.org/licenses/by-nc/3.0/
-# Generated by ./builddist-egi.pl
+# Generated by ./builddist-egi-multi.pl
```

Comment:
Changes in this file are OK. If the addition of debug is controversial/needs discussion then perhaps it should be a separate commit to allow it to proceed without being blocked by the questions above?